With the simple conversion mentioned above I'm getting a build failure based on dsbulk-docs:

```
$ mvn clean deploy -P release
...
[INFO] --- javadoc:3.2.0:jar (attach-javadocs) @ dsbulk-docs ---                                                                                                                                                                              
[INFO] Skipping javadoc generation                                                                                                                                                                                                            
[INFO]                                                                                                                                                                                                                                        
[INFO] --- jacoco:0.8.5:prepare-agent-integration (integration-tests-coverage) @ dsbulk-docs ---                                                                                                                                              
[INFO] argLine set to -javaagent:/home/mersault/work/maven/repo/org/jacoco/org.jacoco.agent/0.8.5/org.jacoco.agent-0.8.5-runtime.jar=destfile=/home/mersault/work/git/dsbulk/docs/target/jacoco-it.exec,excludes=**/generated/**              
[INFO]                                                                                                                                                                                                                                        
[INFO] --- failsafe:2.22.2:integration-test (short-tests) @ dsbulk-docs ---                                                                                                                                                                   
[INFO] No tests to run.                                                                                                                                                                                                                       
[INFO]                                                                                                                                                                                                                                        
[INFO] --- jacoco:0.8.5:report (report-unit-tests-coverage) @ dsbulk-docs ---                                                                                                                                                                 
[INFO] Skipping JaCoCo execution due to missing execution data file.                                                                                                                                                                          
[INFO]                                                                                                                                                                                                                                        
[INFO] --- jacoco:0.8.5:report-integration (report-integration-tests-coverage) @ dsbulk-docs ---                                                                                                                                              
[INFO] Skipping JaCoCo execution due to missing execution data file.                                                                                                                                                                          
[INFO]                                                                                                                                                                                                                                        
[INFO] --- failsafe:2.22.2:verify (short-tests) @ dsbulk-docs ---                                                                                                                                                                             
[INFO]                                                                                                                                                                                                                                        
[INFO] --- gpg:1.6:sign (sign-artifacts) @ dsbulk-docs ---                                                                                                                                                                                    
[INFO]                                                                                                                                                                                                                                        
[INFO] --- install:2.5.2:install (default-install) @ dsbulk-docs ---                                                                                                                                                                          
[INFO] Skipping artifact installation  
[INFO]                                                                                                                                                                                                                                        
[INFO] --- central-publishing:0.7.0:publish (injected-central-publishing) @ dsbulk-docs ---
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for DataStax Bulk Loader 1.11.1:
[INFO] 
[INFO] DataStax Bulk Loader ............................... SUCCESS [  5.281 s]
[INFO] DataStax Bulk Loader - BOM ......................... SUCCESS [  0.530 s]
[INFO] DataStax Bulk Loader - URL ......................... SUCCESS [  6.016 s]
...
[INFO] DataStax Bulk Loader - Workflow - Count ............ SUCCESS [  2.267 s]
[INFO] DataStax Bulk Loader - Runner ...................... SUCCESS [ 31.157 s]
[INFO] DataStax Bulk Loader - Docs ........................ FAILURE [  2.619 s]
[INFO] DataStax Bulk Loader - Distribution ................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:47 min
[INFO] Finished at: 2026-03-03T14:33:12-06:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.sonatype.central:central-publishing-maven-plugin:0.7.0:publish (injected-central-publishing) on project dsbulk-docs: The packaging for this project did not assign a file to the build artifact -> [Help 1]
```

--- a/docs/pom.xml
+++ b/docs/pom.xml
@@ -105,65 +105,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <groupId>org.jacoco</groupId>
-        <artifactId>jacoco-maven-plugin</artifactId>
-        <executions>
-          <execution>
-            <phase>prepare-package</phase>
-            <goals>
-              <goal>report-aggregate</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
-      <plugin>
-        <artifactId>maven-jar-plugin</artifactId>
-        <!-- http://stackoverflow.com/questions/13218313/unable-to-disable-generation-of-empty-jar-maven-jar-plugin -->
-        <executions>
-          <execution>
-            <id>default-jar</id>
-            <phase>none</phase>
-          </execution>
-        </executions>
-      </plugin>
-      <plugin>
-        <artifactId>maven-source-plugin</artifactId>
-        <configuration>
-          <skipSource>true</skipSource>
-        </configuration>
-      </plugin>
-      <plugin>
-        <artifactId>maven-javadoc-plugin</artifactId>
-        <configuration>
-          <skip>true</skip>
-        </configuration>
-      </plugin>
-      <plugin>
-        <artifactId>maven-install-plugin</artifactId>
-        <configuration>
-          <skip>true</skip>
-        </configuration>
-      </plugin>
-      <plugin>
-        <artifactId>maven-deploy-plugin</artifactId>
-        <configuration>
-          <skip>true</skip>
-        </configuration>
-      </plugin>
-      <plugin>
-        <artifactId>maven-gpg-plugin</artifactId>
-        <configuration>
-          <skip>true</skip>
-        </configuration>
-      </plugin>
-      <plugin>
-        <groupId>org.sonatype.plugins</groupId>
-        <artifactId>nexus-staging-maven-plugin</artifactId>
-        <configuration>
-          <skipNexusStagingDeployMojo>true</skipNexusStagingDeployMojo>
-        </configuration>
-      </plugin>
     </plugins>
   </build>
 </project>

--- a/pom.xml
+++ b/pom.xml
@@ -47,7 +47,6 @@
     <module>executor</module>
     <module>workflow</module>
     <module>runner</module>
-    <module>docs</module>
     <module>distribution</module>
   </modules>
   <properties>

